R/extract.R

```diff
@@ -4514,8 +4514,69 @@ setMethod("extract", signature = className("aftreg", "eha"),
 extract.coxreg <- extract.weibreg
 setMethod("extract", signature = className("coxreg", "eha"),
     definition = extract.coxreg)
+    
+    
+# extension for wls objects (metaSEM package)
+# please report errors to Christoph Riedl at Northeastern University; 
+# e-mail: c.riedl@neu.edu
+extract.wls <- function(model, include.nobs = TRUE, ...) {
```

Comment:
This function was already there, but now the formatting is messed up. Please revert this change.